Sources/Confidence/RemoteResolveConfidenceClient.swift

```diff
@@ -26,14 +26,25 @@ class RemoteConfidenceResolveClient: ConfidenceResolveClient {
     // MARK: Resolver
 
     public func resolve(flags: [String], ctx: ConfidenceStruct) async throws -> ResolvesResult {
+        let ctxNetworkValue = TypeMapper.convert(structure: ctx)
         let request = ResolveFlagsRequest(
             flags: flags.map { "flags/\($0)" },
-            evaluationContext: TypeMapper.convert(structure: ctx),
+            evaluationContext: ctxNetworkValue,
             clientSecret: options.credentials.getSecret(),
             apply: applyOnResolve,
             sdk: Sdk(id: metadata.name, version: metadata.version)
         )
 
+        if let debugLogger = options.getLogger() {
```

Comment:
I think it makes more sense to have this log come as part of the "getFlag" API's, then we could also specify in the URL which flag was requested?

Reply:
I am concerned that an app with dozens/hundreds of flags would spam the logs. Also, our UI handles well this scenario: specifying no flags in the URL makes all the available flags for a client show up and be selectable

Reply:
My main problem is that the log output when applied here does not connect very well to when the invocation happens.

Reply:
> I am concerned that an app with dozens/hundreds of flags would spam the logs.
I don't see this as a big problem 😸  -- If the customer things so they would just disable the debuglogger?